### DISCUSSION TITLE
TTGO LoRa32 ESP32 1.6 Display off

### DISCUSSION THREAD

**Discussion by pagex** [lora-aprs/LoRa_APRS_iGate#377] [Q&A]
Wondering why my iGate display is always off, despite unit working fine, in config allways_on  is true, display worked fine when Meshtastic firmware was on that board.

**Comment by gr0mit** [comment]
Hello 
I had to change the file:
../variants/tlora_v2/platformio.ini
the line to (it was v1 before):  
board = ttgo-lora32-v2

Then the ARPS-IS Connection and the Display where working
I did not test the radio yet. 

vy 73
